Makefile

```diff
@@ -79,6 +79,7 @@ export TIKV_BUILD_TIME := $(shell date -u '+%Y-%m-%d %I:%M:%S')
 export TIKV_BUILD_GIT_HASH := $(shell git rev-parse HEAD 2> /dev/null || echo ${BUILD_INFO_GIT_FALLBACK})
 export TIKV_BUILD_GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2> /dev/null || echo ${BUILD_INFO_GIT_FALLBACK})
 export TIKV_BUILD_RUSTC_VERSION := $(shell rustc --version 2> /dev/null || echo ${BUILD_INFO_RUSTC_FALLBACK})
+export TIKV_LOG_TARGETS := $(shell find . -name Cargo.toml -exec awk -F ' ' '/name/{gsub(/"/, "", $$3); print $$3};' {} \; | uniq | xargs)
```

Comment:
This means `cargo build` will no longer build a *useful* executable any more, which is very unfriendly.

Reply:
why?

Reply:
Since `cargo build` will not have `TIKV_LOG_TARGETS` variables set, and result in a TiKV that prints nothing.

Reply:
Oh, you mean build without `make`, that's a problem! maybe use build script?